Sources/SWBUtil/Path.swift

```diff
@@ -781,7 +810,7 @@ public struct Path: Serializable, Sendable {
         }
 
         // Create a string from the first index we found to the end of the path.
-        let trailingSubpath = String(self.str[first..<self.str.endIndex])
```

Comment:
This just got converted back to a Substring immediately in the callee, no reason not to leave it as one to begin with.